Azure Email Communication Services (Lombiq Technologies: OCORE-129) (#14749)
OrchardCMS/OrchardCore: src/docs/releases/1.9.0.md

## Breaking Changes

### Email Module

> Now, enabling `OrchardCore.Email` is only providing an email settings. So you should enable at least one email provider to be able to use the email functionality.

Reviewer: This is still necessary.

> Now, enabling `OrchardCore.Email` is only providing an email settings. So you should enable at least one email provider to be able to use the email functionality.

Author: While everything is explained in the configuration section in `OC.Email` docs I don't think there's a value to add it again

> Now, enabling `OrchardCore.Email` is only providing an email settings. So you should enable at least one email provider to be able to use the email functionality.

Reviewer: If I as a user read the release notes than I won't know that. Secondly, migrating is not the same as configuring the module anew (you have to make note of previous configs and copy them to site settings or rename the config keys), so docs for that is necessary.

> Now, enabling `OrchardCore.Email` is only providing an email settings. So you should enable at least one email provider to be able to use the email functionality.

Reviewer: That's not what I meant. Explain what the user has to do if they already have the Email module installed and configured:

- What should they do if they have config in site settings: e.g., copy settings before upgrade, upgrade, re-configure settings.
- What should they do if they have config in config providers: rename keys.

> Now, enabling `OrchardCore.Email` is only providing an email settings. So you should enable at least one email provider to be able to use the email functionality.

Author: Done!!

### Email Module

> The `OrchardCore.Email` module now only provides email settings and the basics of the email infrastructure and can't be enabled on itself. To actually send out e-mails, you need to enable an email provider, like `OrchardCore.Email.Smtp` or the new `OrchardCore.Email.Azure` (see notes below). The new `OrchardCore.Email.Smtp` module provides the same capabilities previously built into the `OrchardCore.Email` module, so for simple SMTP emails, enable that.
> 
> For the modules author who are using `ISmtpService` service previously, you should now implement `IEmailDeliveryService` instead.

Reviewer: ```suggestion
The `OrchardCore.Email` module now only provides email settings and the basics of the email infrastructure, and can't be enabled in itself. To actually send out e-mails, you need to enable an email provider, like `OrchardCore.Email.Smtp` or the new `OrchardCore.Email.Azure` (see notes below). The new `OrchardCore.Email.Smtp` module provides the same capabilities previously built into the `OrchardCore.Email` module, so for simple SMTP emails, enable that.

Module authors who were using `ISmtpService` service previously should now use `IEmailDeliveryService` instead.
```
